### PR TITLE
browse series view sorting

### PR DESCRIPTION
**Pull request by pgandhizaizi**
## Changes in this PR
added a sorting functionality for browse series view

1. update logic for browse series view sorting in queries.py, 
2. updated example usage in routes.py, 
3. added new test cases for sorting for browse series view in test_queries.py

## JIRA ticket

AYR-600

### After

- [ ] Requires env variable(s) to be updated

### CODE REVIEW

File: app/main/db/queries.py [hunk 1]
```diff
@@ -181,17 +183,16 @@ def _build_transferring_body_view_query(transferring_body_id):
...
         db.session.query(
             Body.BodyId.label("transferring_body_id"),
             Body.Name.label("transferring_body"),
             Series.SeriesId.label("series_id"),
             Series.Name.label("series"),
-            func.to_char(
-                func.cast(func.max(Consignment.TransferCompleteDatetime), DATE),
-                current_app.config["DEFAULT_DATE_FORMAT"],
-            ).label("last_record_transferred"),
+            func.max(Consignment.TransferCompleteDatetime).label(
```
Comment (anthonyhashemi) [inline app/main/db/queries.py:193]: without this change does the sorting not work as expected due to the format of the date?

File: app/main/routes.py [hunk 2]
```diff
@@ -133,6 +133,11 @@ def browse():
     elif series_id:
         browse_type = "series"
         browse_parameters["series_id"] = series_id
+        # sorting_orders["last_record_transferred"] = "asc"  # oldest first
```
Comment (anthonyhashemi) [inline app/main/routes.py:136]: so for series we just sort by last record transferred date and num records held. Cool

**Review by anthonyhashemi — APPROVED** [review]

File: app/main/routes.py (hunk 2 above)
Comment (pgandhizaizi) [reply app/main/routes.py:136]: yes, that is was requested as per design by Terry

File: app/main/db/queries.py (hunk 1 above)
Comment (pgandhizaizi) [reply app/main/db/queries.py:193]: Yes, with DD/MM/YYYY format does not allow correct sorting as postgres DB support yyyy-mm-dd as a default format

**Pull request merged**